test/sqlite/sqllogic_test_runner.cpp

```diff
@@ -112,6 +112,15 @@ void SQLLogicTestRunner::EndLoop() {
 	}
 }
 
+ExtensionLoadResult SQLLogicTestRunner::LoadExtension(DuckDB &db, const std::string &extension) {
+	Connection con(db);
+	auto result = con.Query("LOAD " + extension);
```

Comment:
This change means that every single test we run by default runs into at least one exception ("core_functions" failed to load). Can we only run `LOAD ...` when autoloading is set to a non-default value, instead of enabling this behavior by default?